react-frontend/src/components/Nav/breadcrumbs.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useLocation, useHistory } from 'react-router-dom';
+import '../../css/breadcrumbs.css';
+import { DisplayNames as routeDisplayNames } from '../Nav/routes';
+
+const BreadCrumbs = () => {
+  const history = useHistory();
+  const routeLocation = useLocation();
+  const [pathName, setPathName] = useState(routeLocation.pathname);
+
+  useEffect(() => {
+    history.listen((location) => {
+      console.log('change');
+      setPathName(location.pathname);
+    });
+  }, [history]);
+
+  if (pathName === '/') {
```

Comment:
i'm still not sure why we are setting pathname if it already exists in `routeLocation`